src/test/java/org/apache/commons/lang3/JavaVersionTest.java

```diff
@@ -20,23 +20,10 @@
 
 import org.junit.jupiter.api.Test;
 
-import static org.apache.commons.lang3.JavaVersion.JAVA_RECENT;
+import static org.apache.commons.lang3.JavaVersion.*;
```

Comment:
Checkstyle ain't happy with this .* import I think.

Reply:
Right, we need specific imports.

Reply:
IntelliJ does that by default at greater than 5 import statements. It rewrites them with the *

Reply:
Damn you, IntelliJ! 😤Will fix that later today.